### ISSUE TITLE
Changing video mode does not reload things under new mode

### ISSUE THREAD

**Issue by CarlFK** [mltframework/shotcut#156]
Seems when I use Settings, Video Mode, 720p on a 1080i input, Shotcut does some math and cuts the length of my time line in half... sort of.

The screen shot below is the best I can do for evidence there is a bug and not just me not understanding what is going on.    

If you need more details, please ask and I'll dig more. maybe what I report here will be enough.  

Settings, Video Mode, 720p (not sure if this needs to be done first.)
Open mlt file that references a 1080i file.  
move the playhead to the end.  notice it is 32:20.0 / 32:20.0
Settings, Video Mode, 720p (there is still a * next to that, select it again)
notice we are now at 16:09.0 / 32:20.0 but the play head did not move.  (see screen shot below)
Save, Exit, restart, load file. 
 Now my timeline is only 16:09.0 
even the static png that was at the end has been moved from 32 to 16.

There may be some corrupt metadata in the file because I should be hitting this on 12 other videos I am trying to produce with files from the same source, but the rest seem to be OK.  

![sc1204a](https://cloud.githubusercontent.com/assets/23225/11604795/c9190fa8-9ab6-11e5-8901-79b17a1c4ab8.png)

**Comment by ddennedy** [comment]
I have no idea what the fuck you are talking about. IMO, this is invalid at "Open mlt file that references a 1080i file." Describe the steps using only shotcut or this is closed. You are really getting annoying.

**Comment by CarlFK** [comment]
ok, forget the .mlt, this can be done easy.

Doesn't seem to matter what the starting Video Mode is - I tried Auto, 1080i, 720p 

wget  http://5cda49ca88af98bf1f1e-b4c3b47b38bb1b572e0805ecabeeb59c.r76.cf2.rackcdn.com/SC1ATK104.mov
(53 meg, see specs below)

carl@twist:~/temp$ ~/local/Shotcut/Shotcut.app/shotcut

^o 
pick SC1ATK104.mov

Wait 2 seconds for the play head to get to the end
Settings, Video Mode, 720p 60fps
hit Left Arrow to back up a frame.
see position go from 00:00:02;22 to 00:00:01:21

And now even though the length still says 2;22 the playhead won't move past 1:22



```
carl@twist:~/temp$ mediainfo SC1ATK104.mov 
General
Complete name                            : SC1ATK104.mov
Format                                   : MPEG-4
Format profile                           : QuickTime
Codec ID                                 : qt  
File size                                : 52.3 MiB
Duration                                 : 2s 769ms
Overall bit rate                         : 158 Mbps
Encoded date                             : UTC 2015-11-14 06:09:27
Tagged date                              : UTC 2015-11-14 06:09:27

Video
ID                                       : 9
Format                                   : ProRes
Format version                           : Version 0
Codec ID                                 : apcn
Duration                                 : 2s 769ms
Bit rate mode                            : Constant
Bit rate                                 : 149 Mbps
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 29.970 fps
Color space                              : YUV
Chroma subsampling                       : 4:2:2
Scan type                                : Interlaced
Scan type, store method                  : Interleaved fields
Scan order                               : Top Field First
Bits/(Pixel*Frame)                       : 2.399
Stream size                              : 49.2 MiB (94%)
Writing library                          : AJA Kona Hardware
Language                                 : English
Encoded date                             : UTC 2015-11-14 06:09:27
Tagged date                              : UTC 2015-11-14 06:09:27
```

**Comment by ddennedy** [comment]
Yes, at this time, if you change Video Mode after something is open you need to reload.

**Comment by ddennedy** [comment]
Related to this, there is a major regression in v15.12 that causes the video mode to implicitly become automatic (not shown in UI) when the app starts up and you do not restore a crashed session.

**Comment by ddennedy** [comment]
The regression in v15.12 fixed in a1080d8.
